src/main/kotlin/net/atos/zac/app/admin/FormioFormulierenRestService.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Dimpact
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.zac.app.admin
+
+import jakarta.inject.Inject
+import jakarta.inject.Singleton
+import jakarta.ws.rs.Consumes
+import jakarta.ws.rs.DELETE
+import jakarta.ws.rs.GET
+import jakarta.ws.rs.POST
+import jakarta.ws.rs.Path
+import jakarta.ws.rs.PathParam
+import jakarta.ws.rs.Produces
+import jakarta.ws.rs.core.MediaType
+import jakarta.ws.rs.core.Response
+import net.atos.zac.app.admin.model.RestFormioFormulier
+import net.atos.zac.app.admin.model.RestFormioFormulierContent
+import net.atos.zac.formio.FormioService
+import net.atos.zac.policy.PolicyService
+import nl.lifely.zac.util.NoArgConstructor
+
+@Singleton
+@Path("formioformulieren")
```

Comment:
```suggestion
@Path("formio-formulieren")
```

as in https://restfulapi.net/resource-naming/ section 2.2.3

Reply:
accepted suggestion